app/views/draft_mailer/tool_draft_published_notification.html.erb

```diff
@@ -0,0 +1,22 @@
+
+<% if @is_update %>
+  <h1 class="styled-heading"><%= "#{@short_name} Updated" %></h1>
+<% else %>
+  <h1 class="styled-heading"><%= "#{@short_name} Published" %></h1>
+<% end %>
+
+<div class="email-message">
+  <p>
+    <%= @user[:name] %>,
+  </p>
+  <p>
+    <%= "Your tool metadata record #{@short_name} has been successfully #{ @is_update ? 'updated in' : 'published to'} the CMR#{@email_env_note}. Your tool's concept ID is #{@concept_id}." %>
```

Comment:
If this text is the same in both html and text, do we want to generate it in the mailer?

Reply:
I will take a look into both your mailer comments

Reply:
that's a good point unless you plan to make the concept_id or short_name linkable in the HTML version